flow-model-generator/modules/flow-model-generator-ls-extension/src/main/java/io/ballerina/flowmodelgenerator/extension/AgentsManagerService.java

```diff
@@ -67,12 +73,30 @@ public Class<?> getRemoteInterface() {
         return null;
     }
 
+    @JsonRequest
+    public CompletableFuture<GetAgentOrgResponse> getAgentOrg(GetAgentOrgRequest request) {
+        return CompletableFuture.supplyAsync(() -> {
+            GetAgentOrgResponse response = new GetAgentOrgResponse();
+            try {
+                Path projectPath = Path.of(request.projectPath());
+                Project project = this.workspaceManager.loadProject(projectPath);
+                BLangPackage bLangPackage =
+                        PackageUtil.getCompilation(project.currentPackage()).defaultModuleBLangPackage();
+                response.setOrg(importExists(bLangPackage, BALLERINAX, AI_AGENT) ? BALLERINAX : BALLERINA);
```

Comment:
Why are we using the BLangPackage? Can't we use the `Module` and `SyntaxTree` for this?

Reply:
We can, but It complicates things. We have to loop each document by document and check the importExists